source/filebrowser.cpp

```diff
@@ -489,7 +489,7 @@ int WiiFileLoader()
 
 	if(isBSX()) {
 		sprintf (filepath, "%s%s/BS-X.bin", pathPrefix[GCSettings.LoadMethod], APPFOLDER);
-		if(LoadFile ((char *)Memory.BIOSROM, filepath, 0, 0x100000, SILENT) == 0) {
+		if(LoadFile ((char *)Memory.BIOSROM, filepath, 0, Memory.MAX_ROM_SIZE, SILENT) == 0) {
```

Comment:
Looks like the BSX bios is 0x100000 in size (according to the bsx code). So I think that's the value we should pass in.